src/App.vue

```diff
@@ -31,5 +31,6 @@ export default {
 nav {
   position: absolute;
   background: rgba(0, 0, 0, 0.2);
+  /* display: none; */
```

Comment:
@danielkull : soll ich einmal cleanen, bevor Du es approvest?

Reply:
das bitte noch drin lassen , ist noch in arbeit und muss die manchmal aktivierren